### ISSUE TITLE
Undefined mags, masses when tage=tburst

### ISSUE THREAD

**Issue by rrgupta** [dfm/python-fsps#71]
I don't have a great way to solve this issue, but I just noticed this behavior when playing with adding bursts on top of continuous SFH models:

```
>>> import fsps
>>> sp = fsps.StellarPopulation(sfh=4, tau=2.5, sf_start=1.2, zmet=10, tburst=4, fburst=0.5)
>>> sdss_bands = fsps.find_filter('sdss')
>>> sp.get_mags(bands=sdss_bands, tage=3.99)
 COMPSP WARNING: burst time > age of system.... the burst component will NOT be added.
 COMPSP WARNING: sf_trunc<sf_start.... sf_trunc will be ignored.
array([ 4.64789801,  3.95230007,  3.75145292,  3.8570381 ,  3.59583231])
>>> sp.stellar_mass
0.69857652696034678
>>> sp.get_mags(bands=sdss_bands, tage=4.00)
 COMPSP WARNING: sf_trunc<sf_start.... sf_trunc will be ignored.
array([ nan,  nan,  nan,  nan,  nan])
>>> sp.stellar_mass
nan
>>> sp.get_mags(bands=sdss_bands, tage=4, redshift=0.1)
array([ 99.,  99.,  99.,  99.,  99.])
>>> sp.get_mags(bands=sdss_bands, tage=4.01)
 COMPSP WARNING: sf_trunc<sf_start.... sf_trunc will be ignored.
array([ 2.20035256,  1.9199033 ,  1.47350037,  1.89268185,  0.88935847])
>>> sp.stellar_mass
0.80565726316964659
```

Mags and masses are well-defined just before and just after `tburst` but when `tage=tburst` the mags and masses become `nan` (though if `redshift` is set to a nonzero value in `get_mags(tage=tburst,...)` the mag array returns all `99.` instead of all `nan`). This could be a problem for people who are evaluating CSP mags at discrete time intervals and are interpolating between them for their analyses. 

Since bursts are instantaneous, there will necessarily be discontinuities in the output mags and masses when they occur, but perhaps there is a hacky workaround to at least avoid `nan` and `99.` values which could cause problems. E.g., if the user asks for values at `tage=tburst`, one possible solution is to insert an if-statement to check for this and return values that are actually for `tage=tburst-epsilon` (where `epsilon` is some very small value like 1e-6 or something?) where mags and stellar masses are still defined. There are probably better solutions...What do you think?
Thanks!

**Comment by bd-j** [comment]
Yes, I think this is an issue with how zero-age SSPs are handled within FSPS.  There's probably a place to just output the youngest possible SSP in the case that the burst age is zero (true zero-age isochrones don't exist)

**Comment by bd-j** [comment]
fixed by cconroy20/fsps#17
